mpl_sphinx_theme/__init__.py

```diff
@@ -1,12 +1,14 @@
 from ._version import __version__
 
-from os import path
+from pathlib import Path
+
 
 def get_html_theme_path():
     """Return list of HTML theme paths."""
-    cur_dir = path.abspath(path.dirname(path.dirname(__file__)))
-    return cur_dir
+    return [str(Path(__file__).parent.parent.resolve())]
+
 
-# See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
+# See https://www.sphinx-doc.org/en/master/development/theming.html#distribute-your-theme-as-a-python-package
```

Comment:
Are we working off the dev version?

Reply:
That's where I end up if I go to https://www.sphinx-doc.org/